aws/logs_monitoring/settings.py

```diff
@@ -207,7 +207,14 @@ def __init__(self, name, pattern, placeholder):
         CiphertextBlob=base64.b64decode(ENCRYPTED)
     )["Plaintext"]
     if type(DD_API_KEY) is bytes:
-        DD_API_KEY = DD_API_KEY.decode("utf-8")
+        # If the CiphertextBlob was encrypted with AWS CLI, we
+        # need to re-encode this in base64
```

Comment:
Is this because the AWS CLI encodes it in base64 before encoding in utf-8? It double-encodes it, sort of?

Reply:
Something like that. I couldn't find a reason looking through AWS documentation. 
Basically, if the key itself was first encrypted through the AWS CLI, then decrypting it with the SDK won't work without re-encoding it in b64. We reproduced this behaviour extensively in our Sandbox environment.
See [Plaintext](https://docs.aws.amazon.com/kms/latest/APIReference/API_Decrypt.html#API_Decrypt_ResponseSyntax) in the doc